packages/devextreme/js/__internal/grids/grid_core/column_chooser/m_column_chooser.ts

```diff
@@ -255,8 +252,8 @@ export class ColumnChooserView extends ColumnsView {
       itemTemplate: 'item',
       showCheckBoxesMode: 'none',
       rootValue: null,
-      searchEnabled,
-      searchTimeout,
+      searchEnabled: columnChooser.search?.enabled,
```

Comment:
Minor.

Let's cast the ```searchEnabled``` to the ```boolean```

```suggestion
      searchEnabled: columnChooser.search?.enabled ?? false,
```

Reply:
Not connected to changes in this PR